src/actions/articlesActions.js

```diff
@@ -0,0 +1,13 @@
+import axios from 'axios';
+import GET_ARTICLES from './types';
+
+const fetchArticles = () => dispatch => (
+  axios.get('https://ah-backend.herokuapp.com/api/articles/')
```

Comment:
Do not hardcode the host name in your code. You can take advantage of axios's baseUrl. Also, you are meant to import axios from the helpers folder.